lib/connectors_sdk/confluence/extractor.rb

```diff
@@ -87,7 +87,7 @@ def download(item)
       private
 
       def content_base_url
-        'https://workplace-search.atlassian.net/wiki'
+        config.base_url
```

Comment:
I'd make it `config.base_url || DEFAULT_BASE_URL` ('https://workplace-search.atlassian.net/wiki')

Reply:
What is `DEFAULT_BASE_URL`?